packages/block-library/src/pullquote/edit.js

```diff
@@ -127,7 +127,7 @@ function PullQuoteEdit( {
 						aria-label={ __( 'Pullquote text' ) }
 						placeholder={
 							// translators: placeholder text used for the quote
-							__( 'Write quote…' )
+							__( 'Add quote' )
```

Comment:
Maybe this should be just "Quote"

Reply:
Removing the "add" everywhere makes sense.